.circleci/config.yml

```diff
@@ -61,6 +61,7 @@ jobs:
 
 
   test-core:
+    resource_class: xlarge
```

Comment:
woof

Reply:
Why is this needed now?

Reply:
I'm not sure why these are needed now, but there were what appeared to be timeouts with the smaller class. I don't think that the failures were related to this change - none of the tests relating to this change failed or timed out.

I chose `xlarge` because when I looked, it seemed that that is what we were using before: https://github.com/DataDog/dd-trace-go/commit/435545bd358b5032d8e6e66cf1cab16c6a712a4c#diff-1d37e48f9ceff6d8030570cd36286a61R6

Reply:
I see. We still do, for the contrib. It's fine, but maybe we could use something smaller like `large` and save a bit?

Reply:
Yes, we should open an issue/PR to test with smaller resource classes..